eva-server/pom.xml

```diff
@@ -79,6 +79,11 @@
             <artifactId>guava</artifactId>
             <version>27.0-jre</version>
         </dependency>
+        <dependency>
```

Comment:
I tried making this change, but the response type changed adding some new field and retrieving the null fields as well along with the response probably. I guess I can try to get it right in the next PR since we would be changing the response type by removing QueryResponse and QueryResult there anyway.

Reply:
Do you know in which version of Spring HATEOAS was this fixed? I can see that, if we upgrade to Spring Boot 1.5.7 like other projects, the starter will bring HATEOAS 0.23.0.RELEASE, see [here](http://central.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/1.5.7.RELEASE/spring-boot-dependencies-1.5.7.RELEASE.pom).

Reply:
I will check if it is working with HATEOAS 0.23.0.RELEASE now :)

Reply:
I checked this for `0.23.0.RELEASE` and `0.24.0.RELEASE` versions and it is not working.
But it is working for 0.25.0.RELEASE